src/client/src/rt-platforms/glue/utils/excel.ts

```diff
@@ -71,32 +73,6 @@ const convertBlotterData = (blotterData: BlotterData[]) =>
   blotterData
     .map(data => ({
       ...data,
-      status: `${data.status.charAt(0).toUpperCase()}${data.status.slice(1)}`,
-      tradeDate: convertTradeDate(data.tradeDate),
+      status: _.startCase(_.toLower(data.status)),
```

Comment:
why don't you use _.capitalize?

Reply:
@fdore honestly, I don't want to split hairs here, but the intent was to use title case `startCase` and `_.toLower` converts anything from second character onwards in the string to lower-case in case something is out of place